site/content/en/docs/Examples/custom-controller.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Custom Controller for Agones Game Servers"
+linkTitle: "Custom Controller"
+date:
+publishDate:
+description: >
+  This Custom Controller example shows how to create, deploy and run a Custom Kubernetes Controller for Agones that logs changes to GameServers and modifies their labels.
+---
+
+## Prerequisite
+
+ To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+  
+  - Agones is installed on your cluster. See [Agones guide](https://agones.dev/site/docs/installation/install-agones/).
+
+  - Example custom controller code is available {{< ghlink href="examples/custom-controller" >}}here{{< /ghlink >}}
+
+## Deploy the Custom Controller
+
+For a quick deployment of the custom controller on your cluster, execute:
+
+```bash
+kubectl apply -f deployment.yaml
```

Comment:
Can we do this like we do https://agones.dev/site/docs/getting-started/create-fleet/#1-create-a-fleet ? (point to the public URL) rather than the local file?